src/main/scala/goatrodeo/omnibor/Builder.scala

```diff
@@ -39,11 +39,13 @@ import java.io.FileWriter
 import scala.collection.immutable.TreeSet
 import goatrodeo.util.FileWrapper
 import java.nio.file.Files
+import com.typesafe.scalalogging.Logger
 
 /** Build the GitOIDs the container and all the sub-elements found in the
   * container
   */
 object Builder {
+  val logger = Logger("Builder")
```

Comment:
nit: Any reason not to use `Logger(getClass())` to avoid redundancy here?

Reply:
great suggestion! I copied/pasted other code... _bad dpp... no biscuit for you!_